core/source/finite_element/import_finite_element.cpp

```diff
@@ -1932,7 +2054,7 @@ cmzn_node *EXReader::readNode()
 	}
 	bool result = true;
 	// fill template node if node with identifier already exists (and merge below), otherwise new node
-	FE_node *node = existingNode ? node_template->get_template_node() : returnNode;
+	FE_node *node = existingNode ? nodetemplate->get_template_node() : returnNode;
```

Comment:
Should this be this->nodetemplate->get_template_node() for consistence?

Reply:
Yes - I always use this->member in new code. Will change and pull.